src/menu.tsx

```diff
@@ -45,9 +45,10 @@ export type MenuProps = {
     renderTarget: (props: TargetRenderProps) => React.ReactNode;
     renderMenu: (props: MenuRenderProps) => React.ReactNode;
     children?: void;
+    direction?: 'left' | 'right';
```

Comment:
component API has changed, you should execute `yarn build` and commit updated type definitions